common/lib/xmodule/xmodule/modulestore/tests/test_split_modulestore_bulk_operations.py

```diff
@@ -405,7 +405,13 @@ def test_get_definitions(self, search_ids, active_ids, db_ids):
 
         self.conn.get_definitions.return_value = db_definitions
         results = self.bulk.get_definitions(self.course_key, search_ids)
-        self.conn.get_definitions.assert_called_once_with(list(set(search_ids) - set(active_ids)))
+        definitions_gotten = list(set(search_ids) - set(active_ids))
+        if len(definitions_gotten) > 0:
+            self.conn.get_definitions.assert_called_once_with(definitions_gotten)
+        else:
+            # If no definitions to get, then get_definitions() should *not* have been called.
+            with self.assertRaises(AssertionError):
+                self.conn.get_definitions.assert_called_once_with(definitions_gotten)
```

Comment:
This test will also pass if `get_definitions` is called *more* than once.  Can you assert that it was not called at all?